packages/core/src/modules/article/article-service.ts

```diff
@@ -116,6 +117,10 @@ export class ArticleServiceImpl implements ArticleService {
       .slice(0, 10)
   }
 
+  async getFeatured(): Promise<Article[]> {
+    return (await this.articleRepository.getFeatured()).sort((a, b) => compareDesc(a.updatedAt, b.updatedAt))
```

Comment:
nit: maybe move out the await expression
```suggestion
    const articles = await this.articleRepository.getFeatured()
    return articles.sort((a, b) => compareDesc(a.updatedAt, b.updatedAt))
```